docs/src/content/docs/es/guides/components.mdx

```diff
@@ -173,6 +173,61 @@ import { LinkCard } from '@astrojs/starlight/components';
 	<LinkCard title="Componentes" href="/es/guides/components/" />
 </CardGrid>
 
+### Apartados
+
+Los apartados (también conocidos como "admoniciones" o "llamadas") son útiles para mostrar información secundaria junto al contenido principal de una página.
+
+Un `<Aside>` puede tener un `type` opcional de `note` (el valor predeterminado), `tip`, `caution` o `danger`. Establecer un atributo `title` anula el título predeterminado del aside.
```

Comment:
I think we could standarize to "apartado(s)" since we are using this in the title and opening sentence.

```suggestion
Un `<Aside>` puede tener un `type` opcional de `note` (el valor predeterminado), `tip`, `caution` o `danger`. Establecer un atributo `title` anula el título predeterminado del apartado.
```